bin/generate-dag

```diff
@@ -22,15 +22,14 @@ def generate_dag_fragment(serial, combo):
     contents += '# Node %s: %s - %s - %s\n' % (serial, platform, sources, ', '.join(package_set.packages))
     contents += 'JOB %s single-test-run.sub\n' % (node_name)
     contents += 'VARS %s serial="%s" platform="%s" jobpriority="%d"\n' % (node_name, serial, platform, priority)
-    contents += 'SCRIPT POST %s process-job-output %s $JOBID\n' % (node_name, serial)
+    contents += 'SCRIPT POST %s ../bin/process-job-output %s $JOBID\n' % (node_name, serial)
```

Comment:
Why is this necessary?

Reply:
`process-job-output` lives in `bin/` now with all the other scripts

Reply:
oh and the dag's running from a different directory. got it.